google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryImpl.java

```diff
@@ -237,6 +237,10 @@
   }
 
   private final BigQueryRpc bigQueryRpc;
+  private static final BigQueryRetryConfig DEFAULT_RATE_LIMIT_EXCEEDED_RETRY_CONFIG =
```

Comment:
Should we just name this `DEFAULT_RETRY_CONFIG`? It currently has ratelimitexceeded and we can potentially add more in the future.

Reply:
sure, updated it